src/IndividualArticle.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+const IndividualArticle = () => {
+  const [individualArticle, setIndividualArticle] = useState({});
+  const { article_id } = useParams();
+  console.log(individualArticle);
+  useEffect(() => {
+    axios
```

Comment:
You could separate out your api calls into a separate api.js file, and import them as functions where they're needed 🙂